EIPS/eip-2645.md

```diff
@@ -0,0 +1,68 @@
+---
+eip: 2645
+title: Hierarchical Deterministic Wallet for Computation Integrity Proof (CIP) Layer-2
```

Comment:
Can we shorten this title at all?  This feels quite a bit too long for a title and really would be better as a Simple Summary.

Reply:
Reduced to Hierarchical Deterministic Wallet for Layer-2